ceph-ansible-pipeline/config/definitions/ceph-ansible-pipeline.yml

```diff
@@ -74,11 +77,32 @@
           condition-command: |
             #!/bin/bash
             set -x
-            # if the current branch is master then we don't run these tests
+            # if the target branch is master we DON'T RUN these tests.
             if [[ "$ghprbTargetBranch" == "master" ]]; then
               exit 1
             fi
-            git diff --name-only $(git show HEAD | grep Merge | head -n 1 | cut -d ':' -f2) | grep 'infrastructure-playbooks/purge'
+            git diff --name-only $(git show HEAD | grep Merge | head -n 1 | cut -d ':' -f2) | grep -E 'infrastructure-playbooks/add-osd|tests/functional/centos/7/add-osds'
```

Comment:
you need the same condition than here: https://github.com/ceph/ceph-build/pull/1199/files#diff-95db05e460fbc39f0732490e8119b4fbR63